frontend/src/v5/ui/routes/dashboard/projects/drawings/drawings.component.tsx

```diff
@@ -15,53 +15,96 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { DialogsActionsDispatchers, DrawingsActionsDispatchers } from '@/v5/services/actionsDispatchers';
 import { useEffect } from 'react';
 import { useParams } from 'react-router-dom';
 import { DashboardParams } from '@/v5/ui/routes/routes.constants';
-import { DrawingHooksSelectors } from '@/v5/services/selectorsHooks';
-import AddCircleIcon from '@assets/icons/filled/add_circle-filled.svg';
-import { Button } from '@controls/button';
+import { DrawingsHooksSelectors, ProjectsHooksSelectors } from '@/v5/services/selectorsHooks';
```

Comment:
We are missing the first loading state.

- In drawings it looks like this when fetching the basic drawing list:
<img width="1142" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/cf3f2675-dbb1-473e-b13e-469e708d6a96">

- While in containers it shows this:
<img width="1235" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/cf9240de-7feb-4486-8f07-ceb8eeab3c23">